backend/internal/model/user.go

```diff
@@ -31,7 +32,12 @@ func (u User) WebAuthnID() []byte { return []byte(u.ID) }
 
 func (u User) WebAuthnName() string { return u.Username }
 
-func (u User) WebAuthnDisplayName() string { return u.FirstName + " " + u.LastName }
+func (u User) WebAuthnDisplayName() string {
+	if strings.TrimSpace(u.DisplayName) != "" {
```

Comment:
I recommend not trimming when getting the data. Instread, trim before storing into the dtabase


```suggestion
	if u.DisplayName != "" {
```